packages/babel-cli/src/babel/file.ts

```diff
@@ -70,7 +78,7 @@ export default async function ({
     }
     return count;
   }
-  function emptyMap() {
+  function emptyMap(): Section["map"] {
```

Comment:
This should be exported by trace-mapping directly:
```suggestion
  function emptyMap(): DecodedSourceMap {
```